tasks/minimize_access.yml

```diff
@@ -1,16 +1,25 @@
 ---
-# Using a two-pass approach for checking directories in order to support symlinks.
-- include_tasks: find_files.yml
-  loop_control:
-    loop_var: outer_item
-  loop:
+- name: find files with write-permissions for group
+  shell: "find {{ item }} -perm /go+w -type f"
```

Comment:
There was a comment:
> Using a two-pass approach for checking directories in order to support symlinks

I'm not sure how `find` will work with symlinked directories. I found [this](https://unix.stackexchange.com/a/93858/26279) on StackOverflow. So may be you should use `-L` or `-H` (not sure which one) switch?

Reply:
`-L` seems to go up directories and `-H` does not (https://unix.stackexchange.com/questions/307169/what-is-the-difference-between-find-h-and-find-l-command).

Reply:
I used -L here, thanks! :)